MAUI/DataForm/explicit-data-editors.md

```diff
@@ -0,0 +1,293 @@
+items.Add(new DataFormTextItem() { FieldName = "Name" }); 
+items.Add(new DataFormPasswordItem() { FieldName = "Password" }); 
+items.Add(new DataFormMultilineItem() { FieldName = "Address" }); 
+DataFormGroupItem groupItem = new DataFormGroupItem();
+groupItem.Name = "Address";
+groupItem.Items.Add(new DataFormMultilineItem { FieldName = "Street" });
+groupItem.Items.Add(new DataFormTextItem { FieldName = "State" });
+groupItem.Items.Add(new DataFormTextItem { FieldName = "ZipCode" });
+groupItem.Items.Add(new DataFormAutoCompleteItem { FieldName = "Country", ItemsSource = contactInfoViewModel.CountryNames }); 
+items.Add(groupItem);
+
+dataForm.AutoGenerateItems = false; 
+dataForm.Items = items;
+this.Content = dataForm;
+
+
+
+{% endhighlight %}   
+{% endtabs %} 
+
+{% tabs %}
+{% highlight c# tabtitle="DataFormViewModel.cs" %}
+
+public class DataFormViewModel
+{
+    public DataFormViewModel()
+    {
+        this.ContactDetails = new ContactInfo();
+        this.CountryNames = new List<string>
+        {
+             "United States",
+            "United Kingdom",
+            "France",
+            "Belgium",
+            "Germany"
+        };
+    }
+
+    public ContactInfo ContactDetails { get; set; }
+
+    public IList<string> CountryNames { get; set; }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# tabtitle="ContactInfo.cs" %}
+
+public class ContactInfo
+{
+    public string Name { get; set; }
+
+    public string Password { get; set; }
+
+    public string Street { get; set; }
+
+    public string City { get; set; }
+
+    public string ZipCode { get; set; }
+
+    public string Country { get; set; }
+
+    public string Email { get; set; }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Explicitly add data editor
+
+You can dynamically add the data editor manually by using [Items](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_Items) property of [SfDataForm](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html) when [AutoGenerateItems](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_AutoGenerateItems) is false.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.Items.Add(new DataFormTextItem() { FieldName = "State" });
+this.dataForm.Items.Add(new DataFormTextItem() {FieldName = "Country" });
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Explicitly remove data editor
+You can dynamically remove the data editor explicitly by using the [Items](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_Items) property of [SfDataForm](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html) when [AutoGenerateItems](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_AutoGenerateItems) is false.
```

Comment:
You can dynamically remove the data editor explicitly by using the [Items](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_Items) property of the [SfDataForm](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html) when [AutoGenerateItems](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_AutoGenerateItems) is false.

Reply:
Modified the content